core/embed/models/T3W1/model_T3W1.h

```diff
@@ -91,10 +91,25 @@
 #define ASSETS_SECTOR_END 0x1FF
 
 // RAM layout
-#define KERNEL_U_RAM_SIZE 512
-#define KERNEL_SRAM2_SIZE (64 * 1024 - 512)
+#define FB1_RAM_START 0x30000000
+#define FB1_RAM_SIZE (768 * 1024 - 512)
 
-#define BOOTARGS_SIZE 0x100
+#define BOOTARGS_START 0x300BFE00
+#define BOOTARGS_SIZE 0x200
+
+#define MAIN_RAM_START 0x300C0000
+#define MAIN_RAM_SIZE (64 * 1024 - 512)
+
+#define KERNEL_URAM_START 0x300CFE00
+#define KERNEL_URAM_SIZE 512
+
+#define FB2_RAM_START 0x300D0000
+#define FB2_RAM_SIZE (832 * 1024)
+
+#define AUX_RAM_START 0x301A0000
+#define AUX_RAM_SIZE (832 * 1024)
```

Comment:
Why are FB1 and FB2 not the same size? How about adding 64KB from FB2 to AUX_RAM?